File: Final-Value-of-Variable-After-Performing-Operations/Final-Value-of-Variable-After-Performing-Operations.py
opererations = ["--X","X++","X++"]

def finalValueAfterOperations( operations):
    tong = 0
    for i in range(len(operations)):
        if('+' in operations[i]):
            tong = tong + 1
        else:
            tong = tong - 1
    return tong
    
print(finalValueAfterOperations(opererations))

File: Final-Value-of-Variable-After-Performing-Operations/Final-Value-of-Variable-After-Performing-Operations.js
let operations = ["--X","X++","X++"]


var finalValueAfterOperations = function(operations) {
    let tong = 0
    for(let i = 0;i<operations.length;i++){
        if(operations[i].includes("+")){
            tong++
        }else{
            tong--
        }
    }
    return tong
};

console.log(finalValueAfterOperations(operations))